packages/web_benchmarks/lib/src/browser.dart

```diff
@@ -152,25 +151,26 @@ class Chrome {
     // Subscribe to tracing events prior to calling "Tracing.start". Otherwise,
     // we'll miss tracing data.
     _tracingSubscription =
-        _debugConnection.onNotification.listen((WipEvent event) {
+        _debugConnection?.onNotification.listen((WipEvent event) {
       // We receive data as a sequence of "Tracing.dataCollected" followed by
       // "Tracing.tracingComplete" at the end. Until "Tracing.tracingComplete"
       // is received, the data may be incomplete.
       if (event.method == 'Tracing.tracingComplete') {
-        _tracingCompleter.complete();
-        _tracingSubscription.cancel();
+        _tracingCompleter!.complete();
+        _tracingSubscription!.cancel();
```

Comment:
Why are these safe to force-unwrap?

Reply:
from my perspective, these are safe to unwrap 
_tracingCompleter : declared above
_tracingSubscription is because this is declared when notification listener is created inside which this is called.
but this could be made a conditional call.